src/state/roles/actions.js

```diff
@@ -66,6 +66,7 @@ export const fetchRole = roleCode => dispatch =>
       response.json().then((data) => {
         if (response.ok) {
           dispatch(initialize('role', data.payload));
```

Comment:
do we still need `initialize` ? since redux-form is removed I dont see any reason to call it again?

Reply:
this same action is being used on other pages as well, so I kept it to ensure I wouldn't break anything out of my scope